### PR TITLE
Eat apples

### PR DESCRIPTION
**Pull request by darylkeeter**

### CODE REVIEW

**Review by MichaelFoss — DISMISSED** [review]

**Review by MichaelFoss — CHANGES_REQUESTED** [review]
I think the work done in the server should be componentized. This will allow us to change the method of handling the creation of the apples as well as the eating of the apples without having to interfere with whatever current implementation the rest of the game is in.

The rest of this looks good.

File: src/server.js [hunk 1]
```diff
@@ -71,6 +71,81 @@ const io = sio({
...
+        id: i,
+        color: 'blue',
+        x: getRandomInt(0, 2000),
+        y: getRandomInt(0, 2000)
+    };
+
+    apples.push(apple);
+    appleCollisionCoords = Object.assign({}, appleCollisionCoords, getAppleAreaCoords(apple));
+}
+console.log('FINISHED!');
```
Comment (mAAdhaTTah) [inline src/server.js]: Delete these `console.log` messages. If we need logging, we should look into something like `debug` so it only runs when explicitly called for.

**Review by mAAdhaTTah — CHANGES_REQUESTED** [review]
I agree with @MichaelFoss's suggestion, but we should probably start to seriously consider how we're going to handle state management in the app overall. I'd be ok if we moved those functions out into separate modules and imported them in, and then let me do the refactoring I suggested in your other PR.

I also want to clean up the way webpack is currently integrated on the server side as well, so I can tackle some of this server-side logic.

File: src/server.js (hunk 1 above)
Comment (MichaelFoss) [reply src/server.js]: I'm OK with server-side logs. It's the client-side logs that bother me.

That said, I agree that `debug` is a better approach.

**Comment by MichaelFoss** [comment]
Regarding state management, I couldn't agree more that we should get this correct soon. The closer to the start of the project that we frame the data correctly, the less headaches we'll have in the long run.

That said, I'm not entirely sure it is a reason to fail this PR as it stands. I think the server code should be placed in a separate component, but beyond that I think the other refactoring can be done in a separate PR.

Just my two cents.

**Review by mAAdhaTTah — APPROVED** [review]
I'm good to land this now, but there will definitely be refactoring required later.

**Review by MichaelFoss — APPROVED** [review]

**Pull request merged**